includes/ECommerce.php

```diff
@@ -543,4 +549,81 @@ public function detect_plugin_activation( $plugin ) {
 			}
 		}
 	}
+
+	/**
+	 * Hide Most columns by default
+	 * Shows title and date in the page/post screen by default
+	 *
+	 * @return void
+	 */
+	public function hide_columns() {
+		if( 1 == get_option( 'onboarding_experience_level' ) ) {
+			if ( ! get_user_meta( get_current_user_id(), 'manageedit-pagecolumnshidden' ) ) {
+				update_user_meta( get_current_user_id(), 'manageedit-pagecolumnshidden', array( 'author', 'comments', 'date', 'wpseo-score', 'wpseo-score-readability', 'wpseo-title', 'wpseo-metadesc', 'wpseo-focuskw', 'wpseo-links' ) );
+			}
+			if ( ! get_user_meta( get_current_user_id(), 'manageedit-postcolumnshidden' ) ) {
+				update_user_meta( get_current_user_id(), 'manageedit-postcolumnshidden', array( 'author', 'categories', 'tags', 'comments', 'date', 'wpseo-score', 'wpseo-score-readability', 'wpseo-title', 'wpseo-metadesc', 'wpseo-focuskw', 'wpseo-links' ) );
+			}
+		}
+		
+	}
+
+	/**
+	 * Add custom column header for post/page/product screen
+	 *
+	 * @param array $columns Array of column names for posts/pages
+	 */
+	public function custom_status_column( $columns ) {
+		if ( 'product' != get_post_type() && 1 == get_option( 'onboarding_experience_level' ) ) {
+			// Add 'Status' column after 'Title'
+			$columns['status'] = 'Status';
+		}
+		return $columns;
+	}
+
+	/**
+	 * Shows status and availability under status
+	 *
+	 * @param string $column_name column names to which content needs to be updated
+	 *
+	 * @param int    $post_id Id of post/page
+	 */
+	public function custom_status_column_content( $column_name, $post_id ) {
+		if ( 'status' === $column_name ) {
```

Comment:
Please make it,
``` if (  $column_name  ===  'status')```

Reply:
This too based on the Yoda condition check.